pages/dkp/kommander/2.1/networking/domains/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: layout.pug
+navigationTitle: Domains
+title: Required Domains
+menuWeight: 30
+excerpt: This section describes the required domains for Konvoy
+beta: false
+enterprise: false
+---
+You must have access to the following domains through the customers networking rules so that Konvoy can download all required images:
+
+- k8s.gcr.io
```

Comment:
Do we want these domains alphabetized? Prob doesn't matter too much. Just curious.

Reply:
Might make life easier.